include/openssl/x509.h

```diff
@@ -787,6 +787,14 @@ unsigned long X509_issuer_name_hash_old(X509 *a);
 unsigned long X509_subject_name_hash_old(X509 *x);
 # endif
 
+# define X509_ADD_FLAGS_DEFAULT 0
```

Comment:
Should this be `X509_ADD_FLAG_DEFAULT` (i.e. with no `S`?). None of the others have the `S` so it seems strange that this one does

Reply:
The idea to use the plural for `X509_ADD_FLAGS_DEFAULT` because this macro denotes the whole flags value while the other macros denote just one of the (active) bits.
Yet since this may be more confusing than helpful I've changed the name to `X509_ADD_FLAG_DEFAULT`.